advanced/Scripts/update.sh

```diff
@@ -112,7 +112,7 @@ main() {
     web_update=false
     FTL_update=false
 
-
+    IS_UPDATE=true
```

Comment:
i was attempting to use the `SKIP_INSTALL` variable but it was causing the CentOS tests to fail so I added `IS_UPDATE`.

the idea for this variable is to use it as signal that we are sourcing this function from the update script, which gives an easy dependency check without requiring the reinstall of the meta package every time `pihole -up` is used. 


if there is an update for the `core` repo OR for `FTL` then `basic-install.sh` is started and the variable has no impact.